src/coreclr/vm/typeparse.h

```diff
@@ -401,11 +396,4 @@ class TypeName
     Factory<InlineSString<128> > m_nestNameFactory;
```

Comment:
I plan to delete the unmanaged type name parser and switch VM to use the managed one everywhere in a follow up change.